### PR TITLE
feat(Vukkybox): update presence

### PR DESCRIPTION
**Pull request by nyakase**
**Describe the changes in this pull request:** The presence now no longer displays the user's balance in places where it wouldn't be relevant to do so. This indirectly fixes a bug where the presence breaks if you aren't logged in, as your balance now only appears on pages you need to be logged in to use. In addition, support for guest galleries and the login page has been added.

~~Reminder to self: Lintsweeper is a fun game~~

<details>
<summary> Proof to proving the creation/modification working </summary>
<br>

image
image

</details>

### CODE REVIEW

File: websites/V/Vukkybox/presence.ts [hunk 1]
```diff
@@ -5,62 +5,56 @@ presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
     largeImageKey: "icon",
     smallImageKey: "vukky",
-    smallImageText: "Playing Vukkybox"
+    smallImageText: "Playing Vukkybox",
+    startTimestamp: Date.now()
```
Comment (EncryptedDev) [inline websites/V/Vukkybox/presence.ts:9]: You should make `Date.now()` a varaible

File: websites/V/Vukkybox/presence.ts [hunk 2]
```diff
@@ -5,62 +5,56 @@ presence.on("UpdateData", async () => {
...
   } else if (document.location.pathname === "/") {
-    presenceData.details = "Browsing the Homepage";
-    presenceData.state = `Obtained ${
-      document.getElementById("balance").innerText
-    }!`;
-    presenceData.startTimestamp = Date.now();
+    presenceData.details = "Browsing the homepage";
   } else if (document.location.pathname === "/store") {
-    presenceData.details = "Browsing the Store";
-    presenceData.state = `${
```
Comment (EncryptedDev) [inline websites/V/Vukkybox/presence.ts:25]: Why did you remove this?

File: websites/V/Vukkybox/presence.ts [hunk 3]
```diff
@@ -5,62 +5,56 @@ presence.on("UpdateData", async () => {
...
     presenceData.state = `${
       document.getElementById("balance").innerText
-    } left to spend!`;
+    } Vukkybux left!`;
   } else if (document.location.pathname === "/buyBox/fire") {
     presenceData.details = "Opening a Fire Box";
     presenceData.state = `${
       document.getElementById("balance").innerText
-    } left to spend!`;
+    } lVukkybux left!`;
```
Comment (EncryptedDev) [inline websites/V/Vukkybox/presence.ts:40]: Typo?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Resolve Lint warns too

File: websites/V/Vukkybox/presence.ts (hunk 2 above)
Comment (Vukkyy) [reply websites/V/Vukkybox/presence.ts:25]: Sorry, added back now.

File: websites/V/Vukkybox/dist/metadata.json [hunk 4]
```diff
@@ -4,12 +4,24 @@
...
   },
+  "contributors": [
+    {
+      "name": "Vukky",
+      "id": "708333380525228082"
+    },
+    {
+      "name": "Skelly",
+      "id": "125644326037487616"
+    }
```
Comment (EncryptedDev) [inline websites/V/Vukkybox/dist/metadata.json:15]: Surely not 
image

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**